Forms/MainForm.cs

```diff
@@ -180,7 +183,8 @@ private void detachToolStripMenuItem_Click(object sender, EventArgs e)
 
 		private void newClassToolStripButton_Click(object sender, EventArgs e)
 		{
-			var node = ClassNode.Create();
+			var Address = remoteProcess.IsValid ? remoteProcess.GetModuleByName(remoteProcess.Process.Name).Start : IntPtr.Zero;
```

Comment:
Please use lowercase names (address).
I think you can skip the IsValid call because even if it is valid GetModuleByName could return null. I would suggest something like
`var address = remoteProcess.GetModuleByName(remoteProcess.Process.Name)?.Start ?? IntPtr.Zero;`